### PR TITLE
KUR-44 Implement notifications via the pusher API

### PR DESCRIPTION
**Pull request by milkovsky**
Configure ws server on pusher.com.
Integrate ws server as a pluggable notification channel.
Add a setting for a notification channel plugin.
We should load correct js files(library). Frontend will be also pluggable.
Provide a default notification channel plugin
reloads page after a while in frontend.(frontend)
Create a pusher plugin
Implement notifications via the pusher API. 
We do not spread edit, delete events, only new posts.

### CODE REVIEW

File: src/Form/NotificationChannelSettingsForm.php [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        ];
+      }
+    }
+
+    $form['actions'] = [
+      '#type' => 'actions',
+    ];
+    $form['actions']['submit'] = [
+      '#type' => 'submit',
+      '#value' => $this->t('Submit'),
```
Comment (goldquest) [inline src/Form/NotificationChannelSettingsForm.php]: Can we have the submit button say: Save?

File: modules/liveblog_pusher/src/Plugin/LiveblogNotificationChannel/PusherNotificationChannel.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  function triggerLiveblogPostEvent(LiveblogPost $liveblogPost, $event) {
+    $client = $this->getClient();
+
+    $channel = "liveblog-{$liveblogPost->id()}";
+
+    $data['message'] = 'hello world';
```
Comment (goldquest) [inline modules/liveblog_pusher/src/Plugin/LiveblogNotificationChannel/PusherNotificationChannel.php]: This should be filled with the same data you get via the REST API

File: modules/liveblog_pusher/src/Plugin/LiveblogNotificationChannel/PusherNotificationChannel.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    $form['app_id'] = [
+      '#type' => 'textfield',
+      '#title' => t('App ID'),
+      '#required' => TRUE,
+      '#default_value' => !empty($this->configuration['app_id']) ? $this->configuration['app_id'] : '',
+      '#description' => t('Please enter your Pusher App ID.'),
+    ];
+    $form['key'] = [
+      '#type' => 'textfield',
+      '#title' => t('Username'),
```
Comment (milkovsky) [inline modules/liveblog_pusher/src/Plugin/LiveblogNotificationChannel/PusherNotificationChannel.php]: done

File: src/Form/NotificationChannelSettingsForm.php (hunk 1 above)
Comment (milkovsky) [inline src/Form/NotificationChannelSettingsForm.php]: Done

**Comment by milkovsky** [comment]
Ready for review. I moved the payload method to liveblog post class.

**Pull request merged**